### ISSUE TITLE
Question / Comment: Intended behavior? Annotation.getPixmap with alpha=True returns with background from the page

### ISSUE THREAD

**Issue by PowerSnail** [pymupdf/PyMuPDF#562]
``Annotation.getPixmap`` with ``alpha=True`` still returns the background from page (and other annotations if its in the area).

I'm not sure whether this is a bug, or have I misunderstood the purpose of ``alpha=True``. I would have thought that it returns the annotation rendered in isolation with a transparent background.

Furthermore, if this is intended, is there a way to render the annotation by itself?

PS: Thanks for the work on pymupdf. It's quite pleasant to work with.

A script to reproduce:

```python
import fitz
import sys

doc = fitz.Document(sys.argv[1])
page = doc.loadPage(0)
page.addInkAnnot([[
    (200, 200), (300, 300), (250, 400)
]])
pixmap_page = page.getPixmap(annots=True)
pixmap_page.writeImage("page.png")

annot = next(page.annots())
pixmap_annot = annot.getPixmap(alpha=False)
pixmap_annot.writeImage("annot.png")
```

page.png

![page](https://user-images.githubusercontent.com/7428673/87979851-37fcc500-cac2-11ea-876e-47f813d4d497.jpg)

annot.png

![photo_2020-07-20_12-47-37](https://user-images.githubusercontent.com/7428673/87979893-48ad3b00-cac2-11ea-8b71-37c73aef27a3.jpg)

**Comment by JorjMcKie** [comment]
You got me!
I used to have separate code for annot pixmaps, but introduced the current logic as a workaround for some issues I had at that time.
And the current logic just takes the full page pixmap and clips this down to the annotation rectangle.

Thanks for this input - I will re-introduce the previous code for the next patch.
This should then behave as you expect.
Any urgency for you?

**Comment by PowerSnail** [comment]
Not at all, just doing some explorations. Take your time. 

Thanks!

**Comment by JorjMcKie** [comment]
Resolved by v1.17.4 just uploaded to PyPI.
